src/assets/stylesheets/create-object-dialog.scss

```diff
@@ -0,0 +1,64 @@
+@import 'shared';
+
+:local(.smallButton) {
+  padding-right: 5%;
+  font-size: 2em;
+  align-self: center;
+}
+
+:local(.cancelIcon) {
+  padding-left: 2%;
+  color: white;
+  :hover {
```

Comment:
Should this be `&:hover`? As it is now this is targeting children of something with the `cancelIcon` class that are hovered on.

Reply:
Yes I think so. Woops!